services/static-webserver/client/source/class/osparc/data/model/Node.js

```diff
@@ -882,6 +878,16 @@ qx.Class.define("osparc.data.model.Node", {
     },
     // !---- Input Nodes -----
 
+    // ----- Inputs Required -----
```

Comment:
Q: i thought that qooxdoo could define setters/getters as well as events in the properties. Something like
```javascript
qx.Class.define("YourClassName", {
  extend: qx.core.Object,

  properties: {
    inputsRequired: {
      init: null,
      nullable: true,
      event: "changeInputsRequired",
      apply: "_applyInputsRequired"
    }
  },

  members: {
    _applyInputsRequired: function(value, old) {
      this.__inputsRequired = value;
      // Any additional logic when inputsRequired changes
    }
  }
});
```

Why did you use this approach instead?

Reply:
Because when you use an Array the ```_apply```  will not be triggered when ```set```ting, and it was behaving in a way out of my control.